.github/workflows/cclf-import-test-integration.yml

```diff
@@ -43,9 +47,73 @@ jobs:
 
-  # TODO Check bucket for response file
-  # TODO Run another job to check database for update
+          unzip ../shared_files/cclf/archives/valid2/T.BCD.A0001.ZCY18.D181120.T1000000
+
+          # Clone the synthetic zip file with updated years
+          new_dir="D${date}.T${time}"
+          mkdir $new_dir
+          mv T.BCD.A0001.ZC8Y18.D181120.T1000009 $new_dir/T.BCD.A0001.ZC8Y${year}.D${date}.T${time}1
+          mv T.BCD.A0001.ZC9Y18.D181120.T1000010 $new_dir/T.BCD.A0001.ZC9Y${year}.D${date}.T${time}1
+          mv T.BCD.A0001.ZC0Y18.D181120.T1000011 $new_dir/T.BCD.A0001.ZC0Y${year}.D${date}.T${time}1
+          zip -jr $fname $new_dir
+
+          aws s3 cp --no-progress $fname \
+            s3://bfd-test-eft/bfdeft01/bcda/in/test/$fname
+
+  verify:
+    needs: trigger
+    runs-on: self-hosted
+    env: 
+      ACTIONS_ALLOW_USE_UNSECURE_NODE_VERSION: "true"
+    steps:
+      - uses: actions/checkout@v3
+      - uses: aws-actions/configure-aws-credentials@v3
+        with:
+          aws-region: ${{ vars.AWS_REGION }}
+          role-to-assume: arn:aws:iam::${{ secrets.ACCOUNT_ID }}:role/delegatedadmin/developer/bcda-test-github-actions
+      - name: Install psql
+        run: |
+            sudo amazon-linux-extras install postgresql14
+      - name: Get database credentials
+        uses: cmsgov/ab2d-bcda-dpc-platform/actions/aws-params-env-action@main
+        env:
+          AWS_REGION: ${{ vars.AWS_REGION }}
+        with:
+          params: |
+            CONNECTION_INFO=/bcda/test/api/DATABASE_URL
+      - name: Verify CCLF file was ingested
+        env:
+          FILENAME: ${{needs.trigger.outputs.filename}}
+        # CAUTION: if changing the script below, validate that sensitive information is not printed in the workflow
+        run: |
+          HOST=$(aws rds describe-db-instances --db-instance-identifier bcda-test-rds 2>&1 | jq -r '.DBInstances[0].Endpoint.Address' 2>&1)
+          CONNECTION_URL=$(echo $CONNECTION_INFO 2>&1 | sed -E "s/@.*\/bcda/\@$HOST\/bcda/" 2>&1)
+
+          # Verify that we a record of the CCLF file in the database
+          CCLF_FILE=`psql -t "$CONNECTION_URL" -c "SELECT id FROM cclf_files WHERE name = '$FILENAME' LIMIT 1" 2>&1`
+          if [[ $? -ne 0 || -z $CCLF_FILE ]]; then
+            echo "cclf_file query returned zero results or command failed"
+            exit 1
+          else
+            
+            # Verify that the correct number of benes were imported into the database.
+            CCLF_BENES=`psql -t "$CONNECTION_URL" -c "SELECT count(mbi) FROM cclf_beneficiaries WHERE file_id = $CCLF_FILE" 2>&1`
+              if [[ $? -ne 0 || -z $CCLF_BENES ]]; then
+                echo "CCLF beneficiaries query returned zero results or command failed"
+                exit 1
+              fi
+              if [[ $(echo $CCLF_BENES | xargs) != "6" ]]; then
+                echo "expected 6 beneficiaries imported from file, received $CCLF_BENES".
```

Comment:
is it possible that this will output the stderr here? I know when the connection fails, it can print the connection string.

Reply:
ah, I figured the previous `$?` check would be an indicator that the query was successful (so we'd always be getting some sort of count here.) Do you think that may not be the case?

Reply:
Can remove the "received ..." part here if there's any doubt!

Reply:
oh that's true - if it only hits this condition on success, then I think we are fine!